OctoKit/Releases.swift

```diff
@@ -28,7 +28,7 @@ public struct Release: Codable {
     public let draft: Bool
     public let prerelease: Bool
     public let createdAt: Date
-    public let publishedAt: Date
+    public let publishedAt: Date?
```

Comment:
Can you elaborate why this change was necessary? It would be a breaking change.

Reply:
Same reason as above.